easybuild/easyconfigs/s/SuiteSparse/SuiteSparse-4.5.5-foss-2017a-METIS-5.1.0.eb

```diff
@@ -0,0 +1,21 @@
+name = 'SuiteSparse'
+version = '4.5.5'
+
+homepage = 'http://faculty.cse.tamu.edu/davis/suitesparse.html'
+description = """SuiteSparse is a collection of libraries manipulate sparse matrices."""
+
+toolchain = {'name': 'foss', 'version': '2017a'}
+toolchainopts = {'unroll': True, 'pic': True}
+
+source_urls = ['http://faculty.cse.tamu.edu/davis/SuiteSparse/']
+sources = [SOURCE_TAR_GZ]
+
+metis = 'METIS'
+metis_ver = '5.1.0'
+versionsuffix = '-%s-%s' % (metis, metis_ver)
+dependencies = [(metis, metis_ver)]
```

Comment:
don't use a variable for `'METIS'`, and put the `versionsuffix` right below `version`:

```python
version = '4.5.5'
metis_ver = '5.1.0'
versionsuffix = '-METIS-%s' % metis_ver
...
sources = [SOURCE_TAR_GZ]

dependencies = [('METIS', metis_ver)]